test/integration/serial_test.rb

```diff
@@ -56,5 +68,23 @@ class SerialTest < ActionDispatch::IntegrationTest
       title = find ".serial-content__title"
       assert_equal(@serials.last.title, title.text)
     end
+
+    test "serial description" do
+      serial = create(:serial, :with_posts, title: "serial title", site: @site, description: <<~DESC)
+      # h1
+
+      contents
+      DESC
+      visit "/serials"
+
+      click_on serial.title
+
+      within ".serial-content__description" do
+        h1 = find "h1"
```

Comment:
The above comment insists that we should use the `find` method or other test helpers with parentheses? More descriptions help me to write better codes for our team. 😆